CHANGES.rst

```diff
@@ -317,6 +317,10 @@ astropy.stats
 - Renamed the ``a`` parameter to ``data`` in ``median_absolute_deviation``.
   [#9011]
 
+- Renamed the ``conflevel`` keyword to ``conf`` in ``poisson_conf_interval`` to
```

Comment:
I would have gone the other way around, or even rename both to `confidence_level`, but @larrybradley will have the say of which way to go.

Reply:
Oh... I was pretty sure @astrofrog wanted `conf` but now that I read the issue again, he didn't really say. Ops... I'll change this if there is a consensus.

Reply:
`conf` is ambiguous for me for `config`. But surely when someone is wanting to use a stats function they are not _conf_-used by it that much :)

Reply:
Alright, I think we have a consensus on `confidence_level` as you suggested. Thanks!